script/helpers/PantosHubDeployer.s.sol

```diff
@@ -201,6 +201,13 @@ abstract contract PantosHubDeployer is PantosBaseScript {
 
         address currentPantosToken = pantosHub.getPantosToken();
         if (currentPantosToken != address(pantosToken)) {
+            if (pantosToken.getPantosForwarder() != address(pantosForwarder)) {
+                pantosToken.setPantosForwarder(address(pantosForwarder));
```

Comment:
This looks wrong to me.

We are in the initializePantosHub function, and we are actually performing a state change on the pantosToken. This function should contain only state changes for the PantosHub.